exporter/postgres/queries_common.yml

```diff
@@ -4,6 +4,22 @@
 #
 ###
 
+ccp_pgmonitor_version:
+  query: "SELECT 030300 AS current"
```

Comment:
Can this be populated during build / release ?
I would have extracted this from rpm but they are internal

Reply:
Possibly, but then it won't work for people installing without packages. Open to other solutions.

Reply:
Perhaps something that can run a COPY SYSTEM command to parse the package information out?